test/extension/handlers/topologymutation/handler.go

```diff
@@ -57,16 +57,16 @@ type ExtensionHandlers struct {
 // NewExtensionHandlers returns a new ExtensionHandlers for the topology mutation hook handlers.
 func NewExtensionHandlers() *ExtensionHandlers {
```

Comment:
Let's change this in a way that we handle both v1beta1 & v1beta2 please

(I know right now we don't need v1beta1 anymore, but we're probably going to add more test coverage going forward)